client/wfnews-war/src/main/resources/static.properties

```diff
@@ -11,3 +11,8 @@ wfim-rest.url=https://i1bcwsapi.nrs.gov.bc.ca/wfim-incidents-api
 siteminder.url.prefix=https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?returl=
 bcWildFireSupportPage=https://intranet.gov.bc.ca/bcws/provincial-programs/strategic-initiatives-and-innovation/wildfire-one/wildfire-one-training
 check.token.url=/checkToken.jsp
+webade-oauth2.client.id=WFIM_INCIDENT_MANAGER_UI
+webade-oauth2.wfim-incident-manager-ui.client.secret=${webade-oauth2.wfim-incident-manager-ui.client.secret}
+webade-oauth2.check.token.v2.url=https://intapps.nrs.gov.bc.ca/pub/oauth2/v1/check_token?disableDeveloperFilter=true
```

Comment:
Just a reminder that like the secret token above, we'll have to tokenize all of the configs for environments